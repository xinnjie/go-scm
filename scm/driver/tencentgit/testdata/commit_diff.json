[
    {
        "old_path": "/dev/null",
        "new_path": "README.md",
        "a_mode": 0,
        "b_mode": 33188,
        "diff": "@@ -0,0 +1,2 @@\n+# testme\r\n+a repo for tencentgit client integration test\n\\ No newline at end of file\n",
        "new_file": true,
        "renamed_file": false,
        "deleted_file": false,
        "is_too_large": false,
        "is_collapse": false,
        "additions": 2,
        "deletions": 0
    }
]