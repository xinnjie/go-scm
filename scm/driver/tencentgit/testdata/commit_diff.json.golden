[
    {
        "Path": "README.md",
        "PreviousPath": "/dev/null",
        "Added": true,
        "Renamed": false,
        "Deleted": false,
        "Patch": "@@ -0,0 +1,2 @@\n+# testme\r\n+a repo for tencentgit client integration test\n\\ No newline at end of file\n"
    }
]
